client/src/observatory/index.scss

```diff
@@ -0,0 +1,991 @@
+      margin-bottom: unset;
+    }
+
+    p {
+      color: var(--observatory-color-secondary);
+    }
+
+    form {
+      .input-group {
+        display: flex;
+        height: 3rem;
+
+        :focus-visible {
+          outline: 1px solid var(--observatory-accent);
+          outline-offset: -1px;
+          outline-width: 1px;
+        }
+
+        ::placeholder {
+          color: var(--observatory-color-secondary);
+          opacity: 0.8;
+        }
+
+        input {
+          background-color: var(--observatory-bg);
+          border: 1px solid var(--observatory-border);
+          border-bottom-left-radius: var(--border-radius);
+          border-top-left-radius: var(--border-radius);
+          flex-grow: 1;
+          padding: 0 0.75rem;
+        }
+
+        button {
+          background: var(--button-primary-default);
+          border-bottom-right-radius: var(--border-radius);
+          border-top-right-radius: var(--border-radius);
+          color: var(--background-primary);
+          cursor: pointer;
+          font: var(--type-emphasis-m);
+          font-size: 1rem;
+          padding: 0 2rem;
+
+          &:hover {
+            background: var(--button-primary-hover);
+          }
+
+          &:active {
+            background: var(--button-primary-active);
+          }
+        }
+      }
+
+      label {
+        align-items: center;
+        display: flex;
+        margin: 1.75rem 0;
+      }
+
+      input[type="checkbox"] {
+        height: 1.25rem;
+        margin: 0;
+        margin-right: 0.5rem;
+        width: 1.25rem;
+      }
+    }
+
+    .main {
+      background: var(--observatory-bg);
+      border-radius: var(--border-radius);
+      margin: 1rem 0;
+      // padding: 1rem;
+
+      h2 {
+        margin-top: unset;
+      }
+
+      .about {
+        display: flex;
+        flex-direction: column;
+
+        h2 {
+          margin-bottom: 2rem;
+        }
+
+        svg.lines defs#defs3 #gradient {
+          stop#stop1 {
+            stop-color: var(--observatory-accent);
+          }
+
+          stop#stop2 {
+            stop-color: color-mix(
+              in srgb,
+              var(--observatory-accent),
+              var(--observatory-bg)
+            );
+          }
+
+          stop#stop3 {
+            stop-color: color-mix(
+              in srgb,
+              var(--observatory-accent) 0%,
+              var(--observatory-bg)
+            );
+          }
+        }
+
+        figure {
+          &.assessment,
+          &.scanning,
+          &.security,
+          &.mdn {
+            svg path {
+              fill: var(--observatory-accent);
+            }
+          }
+        }
+      }
+
+      @media (min-width: $screen-md) {
+        .assessment {
+          margin-left: 0;
+        }
+
+        .scanning {
+          margin-left: 3.125rem;
+        }
+
+        .security {
+          margin-left: 6.25rem;
+        }
+
+        .mdn {
+          margin-left: 9.75rem;
+        }
```

Comment:
These should be constant and not move with the viewport width, otherwise the connecting lines graphics would not line up. Even if the lines would scale along, it would not look too good.